js_src/rdm.js

```diff
@@ -5,146 +5,834 @@ angular.module('rdmApp', [])
   $interpolateProvider.endSymbol('}]}');
  }])
 
- .controller('UIDController', ['$scope', function(convertor) {
+ .constant('RDM', {
+  'COMMAND_CLASS': {
+   'DISCOVERY_COMMAND': 16,
```

Comment:
That's the next version - loading the PID information using AJAX. We need to build an index on the server side